awesomplete.js

```diff
@@ -109,6 +109,14 @@ var _ = self.Awesomplete = function (input, o) {
 		inside: this.container
 	});
 	
+	this.status = $.create("span", {
+		hiddena11y: "",
```

Comment:
we *could* use the existing nodes but that would mean that we would need to give them proper focus to have them announced. At the moment the focus never leaves the input and I quite like that. 
Another option to have the selected option announced is to set the content of the input to the selected option. This works fine but would change the current way it works. We *could* have that as an option?
Personally I quite like using the aria-live region as it is the easiest and most universal to use. I bigger projects is very convenient to just have one live-region somewhere and just push messages there when you need to announce something (i.e: "you have new message").